Google/Authorization.php

```diff
@@ -136,8 +137,9 @@ protected function getUserInfoByAccessToken(\Matomo\Dependencies\GoogleAnalytics
     private function getValidUri($uris)
     {
         $validUri = Url::getCurrentUrlWithoutQueryString() . '?module=GoogleAnalyticsImporter&action=processAuthCode';
+        $validUriFallback = SettingsPiwik::getPiwikUrl() . '?module=GoogleAnalyticsImporter&action=processAuthCode'; // Some MWP installs was not working as expected when using Url::getCurrentUrlWithoutQueryString()
```

Comment:
It's doesn't look like `SettingsPiwik::getPiwikUrl()` adds the PHP file, so the fallback URL won't match. I don't have a WP environment to confirm whether this is necessary, but it was when I disabled `$validUri` in my local non-WP instance.
```suggestion
        $validUriFallback = SettingsPiwik::getPiwikUrl() . 'index.php?module=GoogleAnalyticsImporter&action=processAuthCode'; // Some MWP installs was not working as expected when using Url::getCurrentUrlWithoutQueryString()
```

Reply:
@snake14 I did confirm the working with Dizzy and Bojana, as this issue was reproducible for them locally, but I checked my local wp instance after your comment and yes you are correct, we need to add `index.php`

![Screenshot from 2025-05-28 07-17-57](https://github.com/user-attachments/assets/3af51c43-6d49-4778-99d8-a3c546585c14)

Reply:
@snake14 I have asked Dizzy to re-test this locally

Reply:
@snake14 Dizzy suggested to append `index.php` conditionally if not present to cover all the cases